From 6cc65934f9b654cd52e127fb0fa9f54cff7d6efe Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:50 -0400
Subject: [redhat] kabi: Adding symbol ipv6_ext_hdr (net/ipv6/exthdrs_core.c)

Message-id: <24bd3023b75ba5ea78e71c6a1e4e968e1375f874.1410510773.git.poros@redhat.com>
Patchwork-id: 95959
O-Subject: [PATCH RHEL7.1 110/233] kabi: Adding symbol ipv6_ext_hdr (net/ipv6/exthdrs_core.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    To parse IPv6 extension headers. We do support IPv6 parsing to get the flow lookup fields.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/ipv6_ext_hdr
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/ipv6_ext_hdr
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/ipv6_ext_hdr

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/ipv6_ext_hdr b/redhat/kabi/kabi-module/kabi_ppc64/ipv6_ext_hdr
new file mode 100644
index 0000000..a104c89
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/ipv6_ext_hdr
@@ -0,0 +1,2 @@
+#1-
+0x99591a7a ipv6_ext_hdr vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/ipv6_ext_hdr b/redhat/kabi/kabi-module/kabi_s390x/ipv6_ext_hdr
new file mode 100644
index 0000000..a104c89
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/ipv6_ext_hdr
@@ -0,0 +1,2 @@
+#1-
+0x99591a7a ipv6_ext_hdr vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/ipv6_ext_hdr b/redhat/kabi/kabi-module/kabi_x86_64/ipv6_ext_hdr
new file mode 100644
index 0000000..a104c89
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/ipv6_ext_hdr
@@ -0,0 +1,2 @@
+#1-
+0x99591a7a ipv6_ext_hdr vmlinux EXPORT_SYMBOL
-- 
1.7.1